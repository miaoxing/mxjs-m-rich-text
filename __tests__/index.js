import RichText from '..';
import {render} from '@testing-library/react';

describe('RichText', () => {
  test('basic', () => {
    const result = render(<RichText>test</RichText>);
    expect(result.container).toMatchSnapshot();
  });

  test('html', () => {
    const content = '<p>123</p><p><img src="test"/></p>';
    const result = render(<RichText>{content}</RichText>);
    expect(result.container).toMatchSnapshot();
  });
});
