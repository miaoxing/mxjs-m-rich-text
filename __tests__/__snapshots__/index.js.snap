// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RichText basic 1`] = `
<div>
  <taro-view-core
    class="mx-rich-text"
  >
    test
  </taro-view-core>
</div>
`;

exports[`RichText html 1`] = `
<div>
  <taro-view-core
    class="mx-rich-text"
  >
    <p>
      123
    </p>
    <p>
      <img
        src="test"
      />
    </p>
  </taro-view-core>
</div>
`;
